CRM/Civixero/Invoice.php

```diff
@@ -438,15 +438,15 @@ protected function getContributionsRequiringPushUpdate(array $params, int $limit
    *
    * @param array $record
    *
-   * @return array|false
+   * @return array|FALSE
```

Comment:
don't do this - lower case is right for comments

Reply:
Did not know that - ok!